stdlib/string.mli

```diff
@@ -108,14 +117,16 @@ val to_bytes : string -> bytes
     @since 4.13
 *)
 
-external length : string -> int = "%string_length"
-(** [length s] is the length (number of bytes/characters) of [s]. *)
-
-external get : string -> int -> char = "%string_safe_get"
-(** [get s i] is the character at index [i] in [s]. This is the same
-    as writing [s.[i]].
+val blit :
+  string -> int -> bytes -> int -> int -> unit
+(** Rather use {!Bytes.blit_string} but [blit src src_pos ~dst
```

Comment:
The `Rather use ... but ...` wording seems a bit extreme and might fail to fully convey that `Bytes.blit_string` is the same function. Maybe
```
Alternate name for {!Bytes.blit_string}, use directly {!Bytes.blit_string} whenever possible.
[blit src src_pos ~dst len] copies [len] bytes ...
```
?